src/Integrations/Integrations/DatabaseIntegrationHelper.php

```diff
@@ -3,9 +3,15 @@
 namespace DDTrace\Integrations;
 
 use DDTrace\HookData;
+use DDTrace\Tag;
 
 class DatabaseIntegrationHelper
 {
+    public static $PEER_SERVICE_SOURCES = [
```

Comment:
This behaves like a const, so it probably should be a const:
```suggestion
    public const PEER_SERVICE_SOURCES = [
```
(and remove $ signs from usages)